orders/models.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from warehouses.models import Item
+
+
+class Order(models.Model):
+    """
+    Ordered items made by User.
+    """
+    item_id = models.ForeignKey(Item, on_delete=models.CASCADE)
+    date_ordered = models.DateTimeField(auto_now_add=True)
+    order_number = models.CharField(max_length=20)
+    ordered_by = models.ForeignKey(User)
+    is_active = models.BooleanField(default=True)
+
+    class Meta:
+        ordering = ['date_ordered']
+
+    def __str__(self):
+        return self.order_number
+
+
+class OrderDetails(models.Model):
+    """
+    Status of ordered item.
+    """
+    order_id = models.ForeignKey(Order, on_delete=models.CASCADE)
```

Comment:
No need for `_id` here. `order` is understandable enough.

Reply:
Addressed here: https://github.com/jino-d/warehouse/pull/1/commits/59ace3fed4a512764e82158d1ee809b3ee45eeb7